packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -1535,19 +1535,21 @@ mixin WidgetInspectorService {
   }
 
   bool _isValueCreatedByLocalProject(Object value) {
-    final _Location creationLocation = _getCreationLocation(value);
+    final _Location creationLocation = getCreationLocation(value);
     if (creationLocation == null) {
       return false;
     }
     return _isLocalCreationLocation(creationLocation);
   }
 
   bool _isLocalCreationLocation(_Location location) {
-    if (_pubRootDirectories == null || location == null || location.file == null) {
+    if (location == null || location.file == null) {
       return false;
     }
-
     final String file = Uri.parse(location.file).path;
+    if (_pubRootDirectories == null) {
```

Comment:
by default, we check package/flutter

Reply:
make this a comment instead of a note in the code review.
// By default check whether the creation location was within package:flutter.

Reply:
fixed